compositions/upbound-aws-provider/iam-policy/lambda-authorizer-invoke.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
```

Comment:
I don't think this belongs here. Defining Lambda api resources under IAM policies does not feel right to me.

Reply:
Fixed, moved to composition